examples/minimal-egui/src/gui.rs

```diff
@@ -60,7 +60,10 @@ impl Framework {
 
     /// Handle input events from the window manager.
     pub(crate) fn handle_event(&mut self, event: &winit::event::WindowEvent) {
-        self.egui_state.on_event(&self.egui_ctx, event);
+        #[allow(unused_must_use)]
+        {
+            self.egui_state.on_event(&self.egui_ctx, event);
+        }
```

Comment:
It can be ignored by assigning it to a variable:

```suggestion
        let _ = self.egui_state.on_event(&self.egui_ctx, event);
```

In an egui-based application I started working on, I have used the `EventResponse` to determine whether the application needs to repaint.

This example acts more like a game (it needs to repaint every frame) but also doesn't require any keyboard or mouse inputs. [`EventResponse`](https://docs.rs/egui-winit/latest/egui_winit/struct.EventResponse.html) is available for determining if either condition is satisfied. Ignoring the return value is the right thing to do.